matrix-sdk/src/main/java/org/matrix/androidsdk/rest/model/CreateRoomParams.java

```diff
@@ -114,6 +114,12 @@
     @SerializedName("is_direct")
     public Boolean isDirect;
 
+    /**
+     * The power level content to override in the default power level event
+     */
+    @SerializedName("power_level_content_override")
+    public Map<String, Integer> powerLevelContentOverride;
```

Comment:
Why not using type `PowerLevels` instead of a Map?

Reply:
@bmarty you are right but this would need to change the current PowerLevels class (to not be impacted by the default values)
FYI I updated the powerLevelContentOverride type to use the correct one